doc/02-installation.md

```diff
@@ -163,14 +163,13 @@ dnf install https://packages.icinga.com/fedora/icinga-rpm-release-$(. /etc/os-re
     A paid repository subscription is required for SLES repositories. Get more information on
     [icinga.com/subscription](https://icinga.com/subscription)
 
-    Don't forget to fill in the username and password section with your credentials in the local .repo file.
-
 ```bash
 rpm --import https://packages.icinga.com/icinga.key
 
-zypper ar https://packages.icinga.com/subscription/sles/ICINGA-release.repo
+zypper ar -r https://packages.icinga.com/subscription/sles/$releasever/release/ icinga-stable-release
```

Comment:
Where does the variable `$releasever` come from?

Reply:
Well, I found this while looking for the answer: https://doc.opensuse.org/projects/libzypp/HEAD/zypp-repovars.html

It seems like every SuSE-related version uses this variable to determine the version of the OS.

Reply:
Yes, if it would be quoted. Have you tried to provide credentials?

Reply:
@htriem PING

Reply:
I've provided them, still throws some errors. I've decided on a different way of adding the repos more in line with the RHEL way.